README.md

```diff
@@ -112,7 +112,9 @@ Begin with the sample configuration file:
 cp sample-dcrdata.conf dcrdata.conf
 ```
 
-Then edit dcrdata.conf with your dcrd RPC settings. See the output of
+Then edit dcrdata.conf with your dcrd RPC settings. After you are finished, move dcrdata.conf to the ~/.dcrdata/dcrdata.conf folder. 
```

Comment:
Actually the wording is a bit funny as it refers to `~/.dcrdata/dcrdata.conf` as a folder.  Also re-wrap these 3 sentences to it's a bit more nicely formatted.  Thanks for the PR!